com.zettlr.Zettlr.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.zettlr.Zettlr
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+rename-icon: Zettlr
+command: start-zettlr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+  - --filesystem=/tmp
+  - --device=dri
+modules:
+  - name: zettlr
+    buildsystem: simple
+    build-commands:
+      - chmod +x Zettlr-2.1.2-*.AppImage
+      - ./Zettlr-2.1.2-*.AppImage --appimage-extract  
+      - mkdir -p "${FLATPAK_DEST}/zettlr"
+      - cp -r squashfs-root/* "${FLATPAK_DEST}/zettlr"
+      - mv "${FLATPAK_DEST}/zettlr/usr/share/icons" "${FLATPAK_DEST}/share"
+      - install -Dm644 "${FLATPAK_DEST}/zettlr/Zettlr.desktop" "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - desktop-file-edit --set-key=Exec --set-value="start-zettlr" "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm 755 start-zettlr.sh "${FLATPAK_DEST}/bin/start-zettlr"
+      - rm -rf "${FLATPAK_DEST}/share/icons/hicolor/1024x1024/" # a 1024 is too large
```

Comment:
is this really necessary? unless it break the build there is no point removing them

Reply:
yes flatpak-builder errors out saying its to large.

Reply:
exact error message `error: /home/nullrequest/flathub/build-dir/export/share/icons/hicolor/1024x1024/apps/com.zettlr.Zettlr.png is not a valid icon: Image too large (1024x1024). Max. size 512x512`